contracts/deploy/mainnet/111_morpho_wrap_and_transfer.js

```diff
@@ -0,0 +1,52 @@
+const addresses = require("../../utils/addresses");
+const { deploymentWithGovernanceProposal } = require("../../utils/deploy");
+
+module.exports = deploymentWithGovernanceProposal(
+  {
+    deployName: "111_morpho_wrap_and_transfer",
+    forceDeploy: false,
+    // forceSkip: true,
+    reduceQueueTime: true,
+    deployerIsProposer: false,
+    proposalId:
+      "52303668283507532886105041405617076369748861896782994594114630508874108983718",
+  },
+  async () => {
+    const { strategistAddr } = await getNamedAccounts();
+
+    const cLegacyMorpho = await ethers.getContractAt(
+      [
+        "function approve(address,uint256) external",
```

Comment:
this is a nice way of loading an external contract that we will probably never use again